docs/help/_posts/2010-01-01-getting-started.markdown

```diff
@@ -3,7 +3,7 @@ title: Getting started
 layout: post
 ---
 
-## Adding NSubstitute to your test project 
+## [Adding NSubstitute to your test project](#adding-nsubstitute-to-your-test-project)
 
```

Comment:
**question:** Sub-headings are already generated with `id` attributes, is it possible to use those rather than updating the markdown files?

<img width="772" alt="image" src="https://github.com/nsubstitute/NSubstitute/assets/70105/b3ca200d-f7de-478f-b030-3f5ae736beb0">

We probably don't need `h1` anchors as the top url will already go to that.

Reply:
Maybe if there's no `id` it can just copy the page url? (or empty anchor `#`?)
Alternatively if we can get the markdown to generate the h1 ids automatically that'd be great.

I'd just like to avoid manually having to enter ids and match up to the heading contents. 🤔

Reply:
That's already implemented like you suggest:

E.g. "return-for-args"-page has no `hx` in the markdown. It only provides the title to post.html that sets the `h1` using the markdown title.

![image](https://github.com/nsubstitute/NSubstitute/assets/44709506/30555bd3-95b8-4e4d-b974-8cfaa01af42d)
![image](https://github.com/nsubstitute/NSubstitute/assets/44709506/13b1faa9-cbd7-43a6-8c54-78844282601e)


And when there is no `id` in the `hx` the javascript code will just set href to `#`

![image](https://github.com/nsubstitute/NSubstitute/assets/44709506/a00c2876-f594-40b9-a4eb-2b94a38291e4)

So adding a new default post just requires a title as usual which will then be used as page title that reference to the page itself.